### PR TITLE
[Localization] Update French dialogues + typos in general

### PR DESCRIPTION
**Pull request by Adri1**
## What are the changes?
New batch of French translations + typos corrections in several files

## Why am I doing these changes?
To expand French translations and correct typos in several files

## What did change?
New texts in French and typos corrections in several files

## Checklist
- [X] There is no overlap with another PR?
- [X] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?

### CODE REVIEW

File: src/locales/fr/dialogue.ts [hunk 1]
```diff
@@ -28,146 +28,146 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
+      2: "Voyons si t’arrives à garder le rythme !",
+      3: "Accélère le pas, camarade !",
+      4: "J’ai passé 20 ans à la recherche de moi-même…\nMais où suis-je ?"
     },
     "victory": {
-      1: "Tripped up this time!",
-      2: "Oh, I think I'm lost.",
-      3: "Dead end!",
-      4: "Wait up a second! Hey! Don't you know who I am?"
+      1: "J’ai tébuché !",
```
Comment (Dakurei) [inline src/locales/fr/dialogue.ts:139]:
```suggestion
      1: "J’ai trébuché !",
```

File: src/locales/fr/dialogue.ts [hunk 2]
```diff
@@ -28,146 +28,146 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
+      2: "Alors ? Pas mal mes Pokémon, hein ? Je suis fait pour ça.",
+      3: "Peut importe à quel point t’aimes tes Pokémon, il faut toujours de la discipline s’ils se comportent mal."
     }
   },
   "breeder_female": {
     "encounter": {
-      1: "Pokémon never betray you. They return all the love you give them.",
-      2: "Shall I give you a tip for training good Pokémon?",
-      3: "I have raised these very special Pokémon using a special method."
+      1: "Les Pokémon ne tahissent jamais. Ils te rendront toujours l’amour que tu leur donne.",
```
Comment (Dakurei) [inline src/locales/fr/dialogue.ts:80]:
```suggestion
      1: "Les Pokémon ne trahissent jamais. Ils te rendront toujours l’amour que tu leur donne.",
```

File: src/locales/fr/dialogue.ts [hunk 3]
```diff
@@ -28,146 +28,146 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
+      2: "Puis-je te donner une astuce pour mieux élever tes Pokémon ?",
+      3: "J’ai élevé mes Pokémon en utilisant une méthode très spéciale."
     },
     "victory": {
-      1: "Ugh… It wasn't supposed to be like this. Did I administer the wrong blend?",
-      2: "How could that happen to my Pokémon… What are you feeding your Pokémon?",
-      3: "If I lose, that tells you I was just killing time. It doesn't damage my ego at all."
+      1: "Arg… C’était pas supposé se passer comme ça. Leur ai-je administré la mauvaise préparation ?",
+      2: "Comment ça a pu arrver…\nAvec quoi nourris-tu tes Pokémon ?",
```
Comment (Dakurei) [inline src/locales/fr/dialogue.ts:86]:
```suggestion
      2: "Comment ça a pu arriver…\nAvec quoi nourris-tu tes Pokémon ?",
```

File: src/locales/fr/dialogue.ts [hunk 4]
```diff
@@ -28,146 +28,146 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     },
     "victory": {
-      1: "Tripped up this time!",
-      2: "Oh, I think I'm lost.",
-      3: "Dead end!",
-      4: "Wait up a second! Hey! Don't you know who I am?"
+      1: "J’ai tébuché !",
+      2: "Ah, je crois que je me suis paumé.",
+      3: "Ah, une impasse !",
+      4: "Hé ! Attends un seconde…\nTu saurais pas qui je suis ?"
```
Comment (Dakurei) [inline src/locales/fr/dialogue.ts:142]:
```suggestion
      4: "Hé ! Attends une seconde…\nTu saurais pas qui je suis ?"
```

File: src/locales/fr/dialogue.ts [hunk 5]
```diff
@@ -28,146 +28,146 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     },
   },
   "ace_trainer": {
     "encounter": {
-      1: "You seem quite confident.",
-      2: "Your Pokémon… Show them to me…",
-      3: "Because I'm an Ace Trainer, people think I'm strong.",
-      4: "Are you aware of what it takes to be an Ace Trainer?"
+      1: "T’as l’air plutôt confiant.",
+      2: "Tes Pokémon… Monte-les-moi…",
```
Comment (Dakurei) [inline src/locales/fr/dialogue.ts:148]:
```suggestion
      2: "Tes Pokémon… Montre-les-moi…",
```

File: src/locales/fr/dialogue.ts [hunk 6]
```diff
@@ -28,146 +28,146 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
+      4: "OK ! OK ! Tu pourrais être un Topdresseur !"
     },
     "defeat": {
-      1: "I am devoting my body and soul to Pokémon battles!",
-      2: "All within my expectations… Nothing to be surprised about…",
-      3: "I thought I'd grow up to be a frail person who looked like they would break if you squeezed them too hard.",
-      4: "Of course I'm strong and don't lose. It's important that I win gracefully."
+      1: "Je me dévoue corps et âme aux combats Pokémon !",
+      2: "Comme prévu… Vraiment aucune surprise…",
+      3: "Et moi qui pensais qu’en grandissant, j’allais rester frêle et fragile, à me se briser à la moindre étreinte.",
```
Comment (Dakurei) [inline src/locales/fr/dialogue.ts:161]:
```suggestion
      3: "Et moi qui pensais qu’en grandissant, j’allais rester frêle et fragile, à me briser à la moindre étreinte.",
```

File: src/locales/fr/dialogue.ts [hunk 7]
```diff
@@ -2535,146 +2535,146 @@ export const PGFdialogue: DialogueTranslationEntries = {
...
+      2: "Alors ? Pas mal mes Pokémon, hein ? Je suis fait pour ça.",
+      3: "Peut importe à quel point t’aimes tes Pokémon, il faut toujours de la discipline s’ils se comportent mal."
     }
   },
   "breeder_female": {
     "encounter": {
-      1: "Pokémon never betray you. They return all the love you give them.",
-      2: "Shall I give you a tip for training good Pokémon?",
-      3: "I have raised these very special Pokémon using a special method."
+      1: "Les Pokémon ne tahissent jamais. Ils te rendront toujours l’amour que tu leur donne.",
```
Comment (Dakurei) [inline src/locales/fr/dialogue.ts:2587]:
```suggestion
      1: "Les Pokémon ne trahissent jamais. Ils te rendront toujours l’amour que tu leur donne.",
```

File: src/locales/fr/dialogue.ts [hunk 8]
```diff
@@ -2535,146 +2535,146 @@ export const PGFdialogue: DialogueTranslationEntries = {
...
+      2: "Puis-je te donner une astuce pour mieux élever tes Pokémon ?",
+      3: "J’ai élevé mes Pokémon en utilisant une méthode très spéciale."
     },
     "victory": {
-      1: "Ugh… It wasn't supposed to be like this. Did I administer the wrong blend?",
-      2: "How could that happen to my Pokémon… What are you feeding your Pokémon?",
-      3: "If I lose, that tells you I was just killing time. It doesn't damage my ego at all."
+      1: "Arg… C’était pas supposé se passer comme ça. Leur ai-je administré la mauvaise préparation ?",
+      2: "Comment ça a pu arrver…\nAvec quoi nourris-tu tes Pokémon ?",
```
Comment (Dakurei) [inline src/locales/fr/dialogue.ts:2593]:
```suggestion
      2: "Comment ça a pu arriver…\nAvec quoi nourris-tu tes Pokémon ?",
```

File: src/locales/fr/dialogue.ts [hunk 9]
```diff
@@ -2535,146 +2535,146 @@ export const PGFdialogue: DialogueTranslationEntries = {
...
+      2: "Voyons si t’arrives à garder le rythme !",
+      3: "Accélère le pas, camarade !",
+      4: "J’ai passé 20 ans à la recherche de moi-même…\nMais où suis-je ?"
     },
     "victory": {
-      1: "Tripped up this time!",
-      2: "Oh, I think I'm lost.",
-      3: "Dead end!",
-      4: "Wait up a second! Hey! Don't you know who I am?"
+      1: "J’ai tébuché !",
```
Comment (Dakurei) [inline src/locales/fr/dialogue.ts:2646]:
```suggestion
      1: "J’ai trébuché !",
```

File: src/locales/fr/dialogue.ts [hunk 10]
```diff
@@ -2535,146 +2535,146 @@ export const PGFdialogue: DialogueTranslationEntries = {
...
+      2: "Ah, je crois que je me suis paumé.",
+      3: "Ah, une impasse !",
+      4: "Hé ! Attends un seconde…\nTu saurais pas qui je suis ?"
```
Comment (Dakurei) [inline src/locales/fr/dialogue.ts:2649]:
```suggestion
      4: "Hé ! Attends une seconde…\nTu saurais pas qui je suis ?"
```

File: src/locales/fr/dialogue.ts [hunk 11]
```diff
@@ -2535,146 +2535,146 @@ export const PGFdialogue: DialogueTranslationEntries = {
...
     },
   },
   "ace_trainer": {
     "encounter": {
-      1: "You seem quite confident.",
-      2: "Your Pokémon… Show them to me…",
-      3: "Because I'm an Ace Trainer, people think I'm strong.",
-      4: "Are you aware of what it takes to be an Ace Trainer?"
+      1: "T’as l’air plutôt confiante.",
+      2: "Tes Pokémon… Monte-les-moi…",
```
Comment (Dakurei) [inline src/locales/fr/dialogue.ts:2655]:
```suggestion
      2: "Tes Pokémon… Montre-les-moi…",
```

File: src/locales/fr/dialogue.ts [hunk 12]
```diff
@@ -2535,146 +2535,146 @@ export const PGFdialogue: DialogueTranslationEntries = {
...
+      4: "OK ! OK ! Tu pourrais être une Topdresseuse !"
     },
     "defeat": {
-      1: "I am devoting my body and soul to Pokémon battles!",
-      2: "All within my expectations… Nothing to be surprised about…",
-      3: "I thought I'd grow up to be a frail person who looked like they would break if you squeezed them too hard.",
-      4: "Of course I'm strong and don't lose. It's important that I win gracefully."
+      1: "Je me dévoue corps et âme aux combats Pokémon !",
+      2: "Comme prévu… Vraiment aucune surprise…",
+      3: "Et moi qui pensais qu’en grandissant, j’allais rester frêle et fragile, à me se briser à la moindre étreinte.",
```
Comment (Dakurei) [inline src/locales/fr/dialogue.ts:2668]:
```suggestion
      3: "Et moi qui pensais qu’en grandissant, j’allais rester frêle et fragile, à me briser à la moindre étreinte.",
```

**Review by Dakurei — CHANGES_REQUESTED** [review]
Fix typos

**Comment by Adri1** [comment]
> Did you make sure for all of them that they are displayed correctly? And not like cut off because they are too long

I have a scale for this, so unless it's broken it should be okay

**Review by Tempo-anon — APPROVED** [review]

**Pull request merged**